pombola/south_africa/templatetags/za_election_list_links.py

```diff
@@ -0,0 +1,38 @@
+from django import template
+
+register = template.Library()
+
+@register.filter()
+def get_party_slug(org_slug):
+    """Takes the original slug of an election list 'organisation' and returns
+    the party slug
+    """
+
+    stripped = strip_slug(org_slug)
+    parts = stripped.split("--")
+    return parts[0]
+
+@register.filter()
+def get_place_slug(org_slug):
+    """Takes the original slug of an election list 'organisation' and returns
+    the place (province) slug
+    """
+
+    stripped = strip_slug(org_slug)
+    parts = stripped.split("--")
+    return parts[1]
+
+def strip_slug(slug):
```

Comment:
Hope this is ok, I've reworked the regular expression to account for both known slug formats:

```python
election_list_slug_re = re.compile(
    r'''^
    (?P<party>.*)-
    (?P<place_type>provincial|national)-
    (?:(?P<place>.*)-)*
    election-list-
    (?P<year>.*)
    $''',
    re.VERBOSE
)
```

examples:
  `ifp-provincial-western-cape-election-list-2014`
   `ifp-national-election-list-2014`